packages/fluttium/test/src/tester_test.dart

```diff
@@ -302,7 +307,8 @@ void main() {
         semanticsOwner = _MockSemanticsOwner();
 
         final pipelineOwner = _MockPipelineOwner();
-        when(() => binding.pipelineOwner).thenReturn(pipelineOwner);
+        when(() => RendererBinding.instance.rootPipelineOwner)
```

Comment:
```suggestion
        when(() => binding.rootPipelineOwner)
```

You can't mock a non-mocked object. The `WidgetBinding` mock we have has `RendererBinding` mixed into it already